R/utils.R

```diff
@@ -42,98 +42,53 @@
   )
 
   if (is.null(pkgs)) {
-    easystats_pkgs <- .packages_on_github()
-    easystats_on_cran <- .packages_on_cran()
-    easystats_not_on_cran <- setdiff(easystats_pkgs, easystats_on_cran)
+    easystats_pkgs <- easystats_packages()
 
-    local_version <- lapply(easystats_on_cran, utils::packageVersion)
+    local_version <- lapply(easystats_pkgs, utils::packageVersion)
 
     out <- data.frame(
-      package = easystats_on_cran,
+      package = easystats_pkgs,
       cran = NA,
       local = vapply(local_version, as.character, FUN.VALUE = character(1L)),
       behind = FALSE,
       stringsAsFactors = FALSE,
       row.names = NULL
     )
-
-    .add_easystats_dev_pkgs(out, easystats_not_on_cran)
   } else {
-    easystats_pkgs <- .packages_on_github()
-    easystats_on_cran <- intersect(easystats_pkgs, rownames(pkgs))
-    easystats_not_on_cran <- setdiff(easystats_pkgs, easystats_on_cran)
+    easystats_pkgs <- easystats_packages()
 
-    cran_version <- lapply(pkgs[easystats_on_cran, "Version"], package_version)
-    local_version <- lapply(easystats_on_cran, utils::packageVersion)
+    cran_version <- lapply(pkgs[easystats_pkgs, "Version"], package_version)
+    local_version <- lapply(easystats_pkgs, utils::packageVersion)
 
     behind <- unlist(Map(">", cran_version, local_version))
 
     out <- data.frame(
-      package = easystats_on_cran,
+      package = easystats_pkgs,
       cran = vapply(cran_version, as.character, FUN.VALUE = character(1L)),
       local = vapply(local_version, as.character, FUN.VALUE = character(1L)),
       behind = behind,
       stringsAsFactors = FALSE,
       row.names = NULL
     )
-
-    .add_easystats_dev_pkgs(out, easystats_not_on_cran)
-  }
-}
-
-
-.add_easystats_dev_pkgs <- function(out, easystats_not_on_cran) {
```

Comment:
Yeah, I agree about getting rid of this. 

The only reason I had been hesitant to get rid of it thus far was `{circus}`, but that package is not really of any value or relevance to our users anyway.